src/bundler/bundler.rs

```diff
@@ -1,5 +1,37 @@
+use std::str::FromStr;
+
+use clap::Parser;
+use ethereum_types::U256;
```

Comment:
I think we should switch to ethers::type fully, I had some problems with ethereum_types in the past (ethers::types::U256)